src/commands/nickreq/nickreq.ts

```diff
@@ -0,0 +1,71 @@
+const commando = require("discord.js-commando");
+const discord = require("discord.js");
+const { MessageButton, MessageActionRow } = require('discord-buttons');
+let config = require('../../../config.json');
+let db = require("../../db.js");
+
+module.exports = class nickReq extends commando.Command {
+
+	constructor(client) {
+		super(client , {
+			name: 'request',
+			group: 'nickreq',
+			memberName: 'request',
+			description: 'Main Nickname Request commands',
+			argstype: 'single'
+		})
+	}
+	async run(message, args) {
+		if (message.guild == null) return
+		if (args.length > 32) {
+			return await message.reply("The nickname must be less than 32 characters");
+		}
+		if (args.length < 1) {
```

Comment:
Its possible to provide args in the constructor, example:

```ts
      args: [
        {
          key: 'nickname',
          prompt: 'What is the nickname you want to request?',
          type: 'string',
          infinite: true,
        },
      ], 
```

More info here: https://discordjs.guide/commando/args.html#string-arguments